fxmanifest.lua

```diff
@@ -2,7 +2,7 @@ fx_version 'cerulean'
 game 'gta5'
 
 description 'qb-vehicleshop'
-version '2.0.0'
+version '2.0.1'
```

Comment:
Are we ready to version this resource? I was under the impression that we were holding off on beginning that until the Qbox 'release'. Even if we were to version, this would be a major release as it changes the behavior of features, no? https://semver.org

Reply:
rolled back to 2.0.0